### PR TITLE
Update restaurant list

### PR DESCRIPTION
**Pull request by henryymliu**
In light of recent events...

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -1,5 +1,9 @@
 {
     "restaurants": [
+        {
+            "name": "Rosemary Rocksalt",
+            "weight": 9001
```
Comment (soowei) [inline data.json]: This seems a bit high...

Comment (n-martin) [reply data.json]: Agreed. Rosemary Rocksalt is OP. Please nerf.

Comment (henryymliu) [reply data.json]: Would 4 be a more reasonable value?

Comment (n-martin) [reply data.json]: I think 4 would do fine

**Comment by henryymliu** [comment]
I've been a bit busy over the past week, updated it to 4!

File: data.json (hunk 1 above)
Comment (ChristopherzwTan) [reply data.json]: Missed opportunity for a "it's over 9000!" reference. smh.

**Pull request merged**